tests/ui/macro_use_import.stderr

```diff
@@ -0,0 +1,11 @@
+error: `macro_use` attribute's are no longer needed in the Rust 2018 edition
+  --> $DIR/macro_use_import.rs:4:1
+   |
+LL | #[macro_use]
+   | ^^^^^^^^^^^^
+   |
+   = note: `-D clippy::macro-use-import` implied by `-D warnings`
+   = help: remove the attribute and import the macro directly `use std::prelude::macro_name`
```

Comment:
Is this an adequate auto suggestion?

Reply:
No. For a suggestion we'd want to collect the macro invocations and add a `use` entry. Doing that is hard however, so we can for now do without a suggestion.